src/server/Completions/AttributeCompletionFactory.ts

```diff
@@ -44,33 +51,63 @@ export default class AureliaAttributeCompletionFactory extends BaseAttributeComp
 
   /**
    * Look at the View Model and provide all class variables as completion in kebab case.
-   * TODO: Only bindables should be provided.
    */
   private addViewModelBindables(result: CompletionItem[], elementName: string, aureliaApplication: AureliaApplication) {
-    if (aureliaApplication.components.length > 0) {
-      aureliaApplication.components.forEach(component => {
-        if (component.name !== elementName || component.viewModel?.properties === undefined) return;
-
-        component.viewModel.properties.forEach(property => {
-          if (!property.isBindable) return;
-
-          const documentation = `Bindable type: \`${property.type}\``;
-
-          const quote = this.settings.quote;
-          const varAsKebabCase = property.name.split(/(?=[A-Z])/).map(s => s.toLowerCase()).join('-');
-          result.push({
-            documentation: {
-              kind: MarkupKind.Markdown,
-              value: documentation
-            },
-            detail: 'Au View Model Bindable',
-            insertText: `${varAsKebabCase}.$\{1:bind}=${quote}$\{0:${property.name}}${quote}`,
-            insertTextFormat: InsertTextFormat.Snippet,
-            kind: CompletionItemKind.Variable,
-            label: `Au View Model: ${varAsKebabCase}`
+    const program = aureliaApplication.getProgram();
+    if (program !== undefined) {
+      const checker = program.getTypeChecker();
+      for (const sourceFile of program.getSourceFiles()) {
+        const elementNameCandidate = Path.basename(sourceFile.fileName).replace(/\.(ts|js|html)$/, '').split(/(?=[A-Z])/).map(s => s.toLowerCase()).join('-');
+        if (!sourceFile.isDeclarationFile && elementNameCandidate === elementName) {
+          sourceFile.forEachChild((node) => {
+            if (ts.isClassDeclaration(node) && node.name.getText().split(/(?=[A-Z])/).map(s => s.toLowerCase()).join('-') === elementName) {
+              node.members.forEach(member => {
+                if (ts.isPropertyDeclaration(member) && this.propertyIsBindable(member)) {
+                  const propertyName = member.name?.getText();
+                  const bindableType = `Bindable type: \`${member.type?.getText() !== undefined ? member.type?.getText() : "unknown" }\``;
```

Comment:
```suggestion
                  const bindableType = `${member.type?.getText() !== undefined ? member.type?.getText() : "unknown" }\`
                  const bindableTypeText =...
```

Really hard to read otherwise